frontend/src/components/bucket/BucketsTable.vue

```diff
@@ -27,6 +28,10 @@ const showPermissions = async (bucketId: string, bucketName: string) => {
   permissionsBucketId.value = bucketId;
   permissionBucketName.value = bucketName;
 };
+
+const displayPermissionsIcon = (bucket: Bucket) => {
+  return bucket.userPermissions?.find( (x: Permission) => x.permCode === Permissions.MANAGE);
+};
```

Comment:
could consider passing permCode as a param.
later we may want to do `displayIcon(<bucket uuid>, DELETE)`